utils/build.gradle

```diff
@@ -2,6 +2,7 @@ import org.robolectric.gradle.DeployedRoboJavaModulePlugin
 import org.robolectric.gradle.RoboJavaModulePlugin
 
 apply plugin: RoboJavaModulePlugin
+apply plugin: 'kotlin'
```

Comment:
There is one thing I forgot: adding Spotless to this module to let CI checks this module's Kotlin code style.See https://github.com/robolectric/robolectric/wiki/Robolectric's-code-style.

Reply:
oops, thanks for spotting that I overlooked it.